docs/howto/types.md

```diff
@@ -34,6 +33,25 @@ or output expression.
 If a struct contains an embedded struct then SQLair will treat the fields of the
 embedded structs as if they were fields in the parent struct.
 
+### The "omitempty" tag
```

Comment:
On reflection I think this doc should be turned into a reference. If you wanted to use SQLair you would just want to look up how to format your specific type rather than be guided through all the possibilities.

If its ok with you I think I will move this to reference and make it more reference like.

Reply:
I've moved it over, let me know what you think